src/compas_xr/realtime_database.py

```
@@ -0,0 +1,173 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import json
+import pyrebase
+
+
+# Get the current file path
+current_file_path = os.path.abspath(__file__)
+print ("CURRENT FILE PATH = " + current_file_path)
+
+# Define the number of levels to navigate up
+levels_to_go_up = 3
+
+#Construct File path to the correct location
+parent_folder = os.path.abspath(os.path.join(current_file_path, "../" * levels_to_go_up))
+
+# Enter another folder
+target_folder = os.path.join(parent_folder, "scripts")
+print ("TARGET FOLDER PATH = " + target_folder)
+
+#JSON file path
+config_path = os.path.join(target_folder, "firebase_config.json")
+
+# Load the firebase configuration file from the JSON file if it exists: Else: Print.
+if os.path.exists(config_path):
+    with open(config_path) as config_file:
+        print (config_path)
+        config = json.load(config_file)
+
+else:
+    print ("FILE DOES NOT EXIST: " + config_path)
```

if there's an error like this, `raise Exception("File does not exists: " + config_path)` is a better option, otherwise, if you miss the print, your code will keep executing but things won't work correctly (ie. don't mask errors, let them bubble up)